src/scss/_panktee.scss

```diff
@@ -3,10 +3,14 @@
     margin-bottom: -0.25rem;
 
     &.larivaar {
-      word-break: break-all;
+      // word-break: break-all;
```

Comment:
Remove this css block entirely if it's not used.

Reply:
Done